packages/forms-web-app/src/controllers/lpa-dashboard/decided-appeals.js

```diff
@@ -1,16 +1,26 @@
 const { getLPAUserFromSession } = require('../../services/lpa-user.service');
-
+const { getDecidedAppealsCaseDataV2 } = require('../../lib/appeals-api-wrapper');
+const { mapToLPADecidedData } = require('../../lib/dashboard-functions');
 const {
 	VIEW: {
 		LPA_DASHBOARD: { DECIDED_APPEALS, DASHBOARD }
 	}
 } = require('../../lib/views');
 
 const getDecidedAppeals = async (req, res) => {
+	let appealsCaseData = [];
+
 	const user = getLPAUserFromSession(req);
 
+	appealsCaseData = await getDecidedAppealsCaseDataV2(user.lpaCode);
+
+	const decidedAppeals = appealsCaseData.map(mapToLPADecidedData);
+
+	decidedAppeals.sort((a, b) => a.caseDecisionDate - b.caseDecisionDate);
```

Comment:
Does this work? Won't they just be strings?

We could move this to common and use it:
https://github.com/Planning-Inspectorate/appeal-planning-decision/blob/main/packages/web-comment/src/utils/appeal-sorting.js#L47-L66

Reply:
It works at present (js magic?) but I will look at shifting to common as part of the switch to using the api client in common